lib/ansible/modules/windows/win_get_url.ps1

```diff
@@ -23,7 +26,13 @@ $spec = @{
         proxy_username = @{ type='str' }
         proxy_password = @{ type='str'; no_log=$true }
         force = @{ type='bool'; default=$true }
+        checksum = @{ type='str'; aliases=@( 'hash' ) }
+        checksum_algorithm = @{ type='str'; default='sha1'}
```

Comment:
Should we offer a list of choices here ? Or is the list of choices not know in advance ?

Reply:
I guess that the list of choices not know in advance: because we use cmdlet `Get-FileHash` as main function for getting checksumm and params value for algorithm could be changed from version to version.

For ex.: in PS 6 it has only 5 choices (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-filehash?view=powershell-6)

and in PS 5.1 it has 7 choices (https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-filehash?view=powershell-5.1)

Reply:
List of available choices differ for various versions of PS. I'm afraid it's not a good option to set it strictly.

Reply:
Ok, in that case I would probably state this in the parameter documentation.

```
- A list of available options include C(md5), C(sha1), ... For a full list of algorithms look in [the documentation for your version of PowerShell](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-filehash).
```

Reply:
Sounds good. 
Fixed.